### PR TITLE
Readme branch

### PR DESCRIPTION
**Pull request by busraunlu**
I added a README.md to concurrency-examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
+# concurrency-examples
+
+`concurrency-examples` contains some examples about concurrency phenomenon in `GO` language
```
Comment (isacikgoz) [inline README.md]:
```suggestion
`concurrency-examples` contains some examples that are given in several presentations at Google I/O. 
```

**Review by isacikgoz — COMMENTED** [review]
Thanks for the PR @busraunlu! I appreciate the work you've done here. Awesome 👍 

To be more accurate can I ask you to add these `README` files into to folders. It would be more convenient to do so. There would be a `README` file for each example.

Thanks in advance!

File: 2-Generator/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# 2-Generator
+
+In this example's `boring` function is differentiate from first example's. This time, the function returns receive-only channel of strings. The channel is created in the `boring` function and the goroutine is launched in that function. (The same `for` loop as in previous example's `boring` function.)
```
Comment (isacikgoz) [inline 2-Generator/README.md]: Let's give the main idea of this pattern here eg: "Generator pattern is a function that returns a channel of something." Also may be we can add some tips here, such as: "it should or can return a read only `channel`, and by the end of the generation the `channel` can be closed by the generator function itself."

Comment (busraunlu) [reply 2-Generator/README.md]: Thanks a lot for the feedback, now adding these.

**Review by isacikgoz — APPROVED** [review]

**Pull request merged**